guides/common/modules/proc_creating-a-remote-source-for-a-custom-file-type-repository.adoc

```diff
@@ -9,10 +9,10 @@ To create a file type repository in a directory on the base system where {Projec
 
 .Prerequisites
 ifdef::katello,orcharhino[]
-* You have a server running {EL} 9 registered to your {Project}.
+* You have a server running {EL} 9 or {EL} 8 registered to your {Project}.
```

Comment:
```suggestion
* You have a server running {EL} 9 registered to your {Project}.
```

For nightly, Foreman+Katello does not run on EL8 anymore. Foreman 3.12 is the last release that runs on EL8: https://yum.theforeman.org/releases/3.12/ "el8" is missing for "3.13" and nightly.

If you want to only add this to 3.12 and 3.11, then please open a PR against 3.12.